.gitmodules

```diff
@@ -123,8 +123,16 @@ fxDONOTUSEurl = https://github.com/NCAR/ParallelIO
 
 [submodule "doc-builder"]
 path = doc/doc-builder
-url = https://github.com/ESMCI/doc-builder
-fxtag = v1.0.8
+url = https://github.com/samsrabin/doc-builder
+fxtag = c51795f65d230803418c932bfd8639c85c33a19a
```

Comment:
- [ ] Change this back to ESMCI repo, not my fork. Waiting on updates there that (a) [treat sphinx warnings as errors](https://github.com/ESMCI/doc-builder/pull/4) and (b) point to an updated version of the `escomp/base` container that includes `sphinx-mdinclude` (see [this comment](https://github.com/ESCOMP/ESCOMP-Containers/pull/15#issuecomment-2400786134)).